modules/nested-tabs/assets/js/editor/views/view.js

```diff
@@ -18,7 +18,9 @@ export default class View extends $e.components.get( 'nested-elements' ).exports
 			style: '--n-tabs-title-order: ' + index + ';',
 		} );
 
-		if ( 1 === index ) {
+		const isInitialLoad = elementor.previewView.isBuffering;
```

Comment:
We need this, because previously this part of the code was triggered after every repeater action, while we only want this class to be added on page load.